analyze/results_analyze.rb

```diff
@@ -19,7 +19,7 @@
 files = Dir.glob("#{report_directory}/*.report")
 files.each do |file|
   total_time = File.read(file).scan(/Total:\s*((?:\d|\.)+)/)[0][0].to_f
-  ok_responses = File.read(file).scan(/\[OK\]\s*(\d+)/)[0][0].to_f
+  ok_responses = File.read(file).scan(/\[OK\]\s*(\d+)/)[0][0].to_f rescue 0
```

Comment:
The first result when googling `rescue 0 ruby`: Don't Inline-Rescue in Ruby 🥇

Reply:
Yes, the urge for one-liners is not properly represented in best practices. Sadly!